tests/moduletool/test_install.py

```diff
@@ -904,3 +906,86 @@ def test_install_project_with_install_mode_master(tmpdir: py.path.local, snippet
     """.strip() in str(
         excinfo.value
     )
+
+
+@pytest.mark.parametrize_any(
+    "install_module_name, expected_logs, v1_or_v2",
+    [
+        (
+            "minimalv2module",
+            [
+                ("Installing module minimalv2module (v2)", logging.INFO),
+                ("Successfully installed module minimalv2module (v2) version", logging.INFO),
+            ],
+            "v2",
+        ),
+        (
+            "minimalv1module",
+            [("Installing module std (v1)", logging.INFO), ("Successfully installed module std (v1) version", logging.INFO)],
```

Comment:
Why do we check for std in the output?

Reply:
The std module is the only one that get properly installed, the minimalv1module gets copied in the current workdir and so it won't trigger the install logging part. Maybe there's a better way to test for this ?

Reply:
I see. So the std module will get installed for each of the two runs of this test case. Wouldn't it then be better to make this a non-parameterized test case and check for the two log lines at the end of the test case. That would make the test case run faster without losing anything else, right?

Reply:
Indeed :+1: